src/main/js/controls/PairedResult/PairedResult.js

```diff
@@ -188,10 +191,101 @@ class PairedResult extends GraphContent {
      * @inheritDoc
      */
     resize(graph) {
+        const regionItm = [];
+        let localRegionSameFlag = false;
+
         if (utils.notEmpty(this.dataTarget.regions)) {
-            // If graph has more than 1 content then we dont load the regions
+            const regionList = this.dataTarget.regions;
+            const values = this.dataTarget.values;
+
+            //check if all region are there with respect to value
+            if (graph.config.isPairedDataProper === true) {
```

Comment:
In case of a Paired Result its pretty much the same case but an extra step is necessary, currently we are only putting the data of `start`, `end` and `axis` to the regions but along with that you will also put the `key` (most likely [here](https://github.com/cerner/carbon-graphs/blob/master/src/main/js/controls/PairedResult/helpers/helpers.js#L605)). What this will allow is that you will get the context of which region is belonging to what content. 

As always, the logic would be to show the region until one of them is different.

| Content 1 | Content 2 | Content 3 | Result   |
| --------- | --------- | --------- | -------- |
| R1        | R2        | R3        | Hide All |
| R1        | R1        | R2        | Hide All |
| R1        | R1        | R1        | Show     |
| R1        | -         | -         | Hide All |
| R1        | R1        | -         | Hide All |
| -         | -         | -         | N/A      |

Reply:
We can not go with the dom approach  since in the case if region is not provided it will not create the region for it, that is why we switched to this approach.

Reply:
That works even today. We just hide all the regions, at that point. 
https://github.com/cerner/carbon-graphs/blob/master/src/main/js/controls/Line/Line.js#L167

Reply:
its for Paired not for line.